provider/marathon/marathon.go

```diff
@@ -55,7 +55,9 @@ type Provider struct {
 	MarathonLBCompatibility   bool             `description:"Add compatibility with marathon-lb labels" export:"true"`
 	FilterMarathonConstraints bool             `description:"Enable use of Marathon constraints in constraint filtering" export:"true"`
 	TLS                       *types.ClientTLS `description:"Enable TLS support" export:"true"`
-	DialerTimeout             flaeg.Duration   `description:"Set a non-default connection timeout for Marathon" export:"true"`
+	DialerTimeout             flaeg.Duration   `description:"Set a non-default dialer timeout for Marathon" export:"true"`
```

Comment:
Do we need to use the phrase "non-default"? Isn't it kind of assumed that defaults will be used unless specified?

Reply:
Good point, thanks! I removed the "non-default".